lib/SimplePie/SimplePie/Parser.php

```diff
@@ -181,12 +181,20 @@ public function parse(&$data, $encoding, $url = '')
 			xml_set_element_handler($xml, 'tag_open', 'tag_close');
 
 			// Parse!
-			if (!xml_parse($xml, $data, true))
-			{
-				$this->error_code = xml_get_error_code($xml);
-				$this->error_string = xml_error_string($this->error_code);
-				$return = false;
-			}
+			$stream = fopen('php://memory','r+');                             
+			fwrite($stream, $data);                                           
+			rewind($stream);                                                  
+																			  
+			while (($stream_data = fread($stream, 16384))) {                  
```

Comment:
But that part could be sent upstream https://github.com/simplepie/simplepie/blob/98ab0fe9f5c04e72c2015eb2ba6a4bedff2dabd1/library/SimplePie/Parser.php#L167

Reply:
Alright I will push a PR there

Reply:
https://github.com/FreshRSS/FreshRSS/pull/3416